website/www/site/content/en/documentation/sdks/python-machine-learning.md

```diff
@@ -243,6 +243,17 @@ For more information, see the [`PredictionResult` documentation](https://github.
 For detailed instructions explaining how to build and run a Python pipeline that uses ML models, see the
 [Example RunInference API pipelines](https://github.com/apache/beam/tree/master/sdks/python/apache_beam/examples/inference) on GitHub.
 
+## Slowly updating side input pattern to update models used by ModelHandler
```

Comment:
Is slowly updating a type of side input pattern? If so, I think it should be hyphenated:

Slowly-updating side input pattern

Reply:
Changed it.